icons/liara.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><defs><linearGradient id="prefix__a" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.1724 0 0 .1724 -.62 -.587)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient><linearGradient id="prefix__b" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.1724 0 0 .1724 -.62 -.587)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient><linearGradient id="prefix__c" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.1724 0 0 .1724 -.62 -.587)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient></defs><path fill="url(#prefix__a)" d="M8.719 9.668c0-.563.398-1.246.886-1.527l4.176-2.395c.489-.281.887-.055.887.508v4.875c0 .559-.398 1.242-.887 1.523l-4.176 2.399c-.488.281-.886.054-.886-.508zm0 0"/><path fill="url(#prefix__b)" d="M1.332 6.254c0-.563.398-.79.887-.508l4.176 2.395c.488.28.886.964.886 1.527v4.875c0 .562-.398.789-.886.508l-4.176-2.399c-.489-.28-.887-.964-.887-1.527zm0 0"/><path fill="url(#prefix__c)" d="M2.86 4.453c-.489-.281-.489-.738 0-1.016L7.034 1.04c.488-.281 1.281-.281 1.77 0l4.246 2.438c.488.28.488.734 0 1.015L8.875 6.891c-.488.28-1.281.28-1.773 0zm0 0"/></svg>
```

Comment:
This icon will most likely not be pixel-perfect, but I think this option might work

<img src="https://github.com/user-attachments/assets/a6dfa090-f378-4a8e-bae9-5b43ec8e9195" width="300">

```suggestion
<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient id="prefix__a" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.15939 0 0 .16254 -.134 -.857)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient><linearGradient id="prefix__b" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.15939 0 0 .16253 .195 -.856)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient><linearGradient id="prefix__c" gradientUnits="userSpaceOnUse" x1="11.328" y1="8.203" x2="90.301" y2="10.761" gradientTransform="matrix(.15784 0 0 .16493 .15 -.355)"><stop offset="0" stop-color="#69F0AE"/><stop offset="1" stop-color="#4FC3F7"/></linearGradient></defs><path fill="url(#prefix__a)" d="M8.5 8.811c0-.53.368-1.174.82-1.44l3.86-2.257c.452-.265.82-.052.82.479v4.596c0 .527-.368 1.17-.82 1.436l-3.86 2.261c-.452.265-.82.051-.82-.479zm0 0"/><path fill="url(#prefix__b)" d="M2 5.593c0-.53.368-.745.82-.479l3.86 2.258c.452.264.82.909.82 1.44v4.595c0 .53-.368.744-.82.48l-3.86-2.262c-.452-.264-.82-.909-.82-1.44zm0 0"/><path fill="url(#prefix__c)" d="M3.336 4.467c-.448-.27-.448-.706 0-.972l3.821-2.293c.447-.27 1.173-.27 1.62 0l3.888 2.332c.447.268.447.702 0 .971L8.843 6.799c-.447.268-1.173.268-1.624 0zm0 0"/></svg>
```